test/unit/pages/confirm/controller-test.js

```diff
@@ -45,9 +52,14 @@ describe('ConfirmController', function() {
     sandbox.stub(fs, 'existsSync').returns(true);
     sandbox.spy(_$state_, 'go');
     installerDataSvc.setup();
-    for (var installer of installerDataSvc.allInstallables().values()) {
-      sandbox.stub(installer, 'detectExistingInstall').resolves();
-    }
+
+    sandbox.stub(CDKInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(CygwinInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(HypervInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(JdkInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallWindows.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallDarwin.prototype, 'detectExistingInstall').resolves();
```

Comment:
That sounds good, I was just testing if it is possible to move components loading to confirmation page, so I can at some point implement updating requirements.json from remote location.